portable/IAR/ARM_CM4F_MPU/port.c

```diff
@@ -777,6 +785,32 @@ BaseType_t xPortStartScheduler( void )
         configASSERT( portCPUID != portCORTEX_M7_r0p0_ID );
     #endif
 
+    /* Applications that route program control to the FreeRTOS interrupt
+     * handlers through intermediate handlers (indirect routing) should set
+     * configCHECK_HANDLER_INSTALLATION to 0 in FreeRTOSConfig.h. Direct
+     * routing, which is validated here when configCHECK_HANDLER_INSTALLATION
+     * is 1, is preferred when possible. */
+    #if ( configCHECK_HANDLER_INSTALLATION == 1 )
+    {
+        const portISR_t * const pxVectorTable = portSCB_VTOR_REG;
+
+        /* Verify correct installation of the FreeRTOS handlers for SVCall and
+         * PendSV. Do not check the installation of the SysTick handler because
+         * the application may provide the OS tick without using the SysTick
+         * timer by overriding the weak function vPortSetupTimerInterrupt().
+         *
+         * Assertion failures here can be caused by incorrect installation of
+         * the FreeRTOS handlers. For help installing the handlers, see
+         * https://www.FreeRTOS.org/FAQHelp.html
+         *
+         * Systems with a configurable address for the interrupt vector table
+         * can also encounter assertion failures or even system faults here if
+         * VTOR is not set correctly to point to the application's vector table. */
+        configASSERT( pxVectorTable[ portVECTOR_INDEX_SVC ] == vPortSVCHandler );
+        configASSERT( pxVectorTable[ portVECTOR_INDEX_PENDSV ] == xPortPendSVHandler );
+    }
+    #endif /* configCHECK_HANDLER_INSTALLATION */
+
```

Comment:
Missing `portNVIC_SHPR2_REG = 0;` after
```
    portNVIC_SHPR3_REG |= portNVIC_PENDSV_PRI;
    portNVIC_SHPR3_REG |= portNVIC_SYSTICK_PRI;
```

Reply:
Another MPU port -- has its own mechanism for setting SVCall priority.